Tools/Cli-LoRa-Device-Provisioning/LoRaWan.Tools.CLI/DefaultRouterConfig/US902.json

```diff
@@ -42,17 +42,17 @@
         0
       ],
       [
-        0,
+        -1,
         0,
         0
       ],
```

Comment:
We should also update the AU915 to remain consistent:

from RP1 https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf
AU915 : DR7 should be -1

Reply:
Thanks, updated with 685a2a25